src/vector/index.js

```diff
@@ -486,6 +483,10 @@ async function verifyServerConfig() {
 
         if (hsUrl) {
             console.log("Config uses a default_hs_url - constructing a default_server_config using this information");
+            console.warn(
+                "DEPRECATED CONFIG OPTION: In the future, default_hs_url will not be accepted. Please use " +
+                "default_server_config instead.",
+            );
```

Comment:
I would be OK with keeping this as a shorthand, but fine either way.

Reply:
I'm of similar thinking, although also wanting to keep the ways of configuring a homeserver to a minimum. I've hopefully worded the docs to say "it's deprecated, but less deprecated than `default_*_url`"